src/aiokatcp/sensor.py

```diff
@@ -823,23 +838,37 @@ def update_aggregate(self, updated_sensor, reading, old_reading) -> Reading:
         """
         pass
 
+    def filter_aggregate(self, sensor: Sensor) -> bool:
+        """Check that a sensor isn't this aggregate sensor.
```

Comment:
That's the default implementation but not the purpose of the function.

```suggestion
        """Decide whether another sensor is part of the aggregation.
```